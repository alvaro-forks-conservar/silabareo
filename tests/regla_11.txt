fútbol
output
tutti
frutti
etcétera
catchup
etnia
algoritmo