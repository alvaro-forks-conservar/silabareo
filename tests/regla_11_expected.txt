fút-bol
out-put
tut-ti
frut-ti
et-cé-te-ra
cat-chup
et-nia
al-go-rit-mo